src/napari_vedo_bridge/_cutter_widget.py

```diff
@@ -69,6 +74,24 @@ def send_to_napari(self):
 
         self.napari_viewer.add_surface(mesh_tuple)
 
+    def box_cutter_tool(self):
+        """
+        Add a box cutter tool to the vedo plotter
+        """
+        self.vedo_message.text(
+            "Press r to reset the cutting box\n"
+            'Press spacebar to toggle the cutting box on/off\n'
+            'Press i to invert the selection\n',
```

Comment:
Cool! I'd then simply remove the `Invert` checkbox from the UI. Makes it cleaner and less code.

Reply:
Still probably easier to handle than tracking yet another callback function, state flag, etc